projects/plugins/jetpack/3rd-party/3rd-party.php

```diff
@@ -17,6 +17,11 @@
  * This is a refactor of load_3rd_party() to load the individual compat files only when needed instead of universally.
  */
 function load_3rd_party_compat_filters() {
+	// ActivityPub.
+	if ( class_exists( '\Activitypub\Model\Post' ) ) {
```

Comment:
Nitpick, feel free to ignore
```suggestion
	if ( class_exists( \Activitypub\Model\Post::class ) ) {
```